### PR TITLE
bleach

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: recipes/bleach/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - html5lib >=0.999,!=0.9999,!=0.99999,<0.99999999
+    - six
+
+test:
+  imports:
+    - bleach
+
+about:
+  home: https://github.com/mozilla/bleach
+  license: Apache
```
Comment (jakirkham) [inline recipes/bleach/meta.yaml]: What version of Apache?

Also could optionally add `license_family: Apache` IIRC.

File: recipes/bleach/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - html5lib >=0.999,!=0.9999,!=0.99999,<0.99999999
```
Comment (jakirkham) [inline recipes/bleach/meta.yaml]: This is fun. What is going on here? 😕

Comment (minrk) [inline recipes/bleach/meta.yaml]: Yup, this is weird, but it's how html5lib does it - 0.9x approaching 1.0.

[bleach setup.py ref](https://github.com/mozilla/bleach/blob/v1.5/setup.py#L18)

**Comment by jakirkham** [comment]
Thanks @minrk .

**Pull request merged**